src/sentry/receivers/onboarding.py

```diff
@@ -100,11 +100,16 @@ def record_new_project(project, user=None, user_id=None, **kwargs):
         project_id=project.id,
     )
     if not success:
+        organization = Organization.objects.get(id=project.organization_id)
```

Comment:
i don't think this RPC call is needed. I think that you can use `project.organization` to have organization object

Reply:
oh yes i can see the same in the row 91. Good call :)